src/node_bob.h

```diff
@@ -0,0 +1,111 @@
+#ifndef SRC_NODE_BOB_H_
```

Comment:
isn't bob a different kind of stream? its used by QUIC? can it be its own PR, or maybe ifdefed on QUIC being enabled?

Reply:
It is used directly by the QUIC implementation, I'll guard it on QUIC being enabled.